example/lib/main.dart

```diff
@@ -95,6 +97,69 @@ class _SearchFieldSampleState extends State<SearchFieldSample> {
           padding: const EdgeInsets.all(8.0),
           child: ListView(
             children: [
+              UserSelect(),
+              SizedBox(
+                height: 550,
+              ),
+              DynamicHeightExample(),
+              SizedBox(
+                height: 50,
+              ),
+              SearchField(
+                hint: 'Basic SearchField',
+                dynamicHeightItem: true,
+                initialValue: SearchFieldListItem<String>('ABC'),
+                suggestions: [
```

Comment:
Store this suggestions in a variable, so that you don't have to repeat this long list